drivers/clock_control/clock_control_nrf.c

```diff
@@ -14,6 +14,10 @@
 #include <logging/log.h>
 #include <shell/shell.h>
 
+#if CONFIG_SOC_NRF5340_CPUAPP
```

Comment:
This preprocessor guard should also match the one used below, for consistency.
(unless nrf_gpio.h inclusion is needed outside the scope of the added code block in clk_init

Reply:
done